_assets/scripts/migration_check.sh

```diff
@@ -18,9 +18,7 @@ check_migration_order() {
   done
 }
 
-git checkout origin/develop
-git pull origin develop
-git checkout -
+git fetch origin develop
```

Comment:
This wasn't working for some reason, is how we had it before, but wasn't picking up changes

Reply:
That's interesting.. (thanks or heads-up).

Reply:
To re-test this. it should be enough to create "outdated" migration files, right?

Reply:
unfortunately no, since it was working before when I first implmented it, and then it stopped, so I don't think is so easy to replicate (but worth trying, but we can only trust if the error is not picked up, but not if it is)